pkg/entrypoint/entrypointer.go

```diff
@@ -121,6 +123,25 @@ func (e Entrypointer) Go() error {
 	return err
 }
 
+// fmtErrOutPut will format error to PipelineResourceResult
+func (e Entrypointer) fmtErrOutPut(err error) []v1alpha1.PipelineResourceResult {
+	if err == nil {
+		return []v1alpha1.PipelineResourceResult{}
+	}
+	// TODO: will add logic to format reason according error type sooner, example: DeadlineExceeded
```

Comment:
Please reference an issue or doc describing this in more detail in case we stumble upon this again in a year.

Reply:
I still see this TODO without an issue reference.

Reply:
Wow, I referenced the issue in next line original. En... I have added the issue reference in current line in latest commit.